src/lib/containers/CardContainer.tsx

```diff
@@ -55,7 +55,7 @@ export const CardContainer = (props: CardContainerProps) => {
       case FUNDER:
         return <Funder {...props} />
       case GENERIC_CARD:
-        return <GenericCard {...props} />
+        return <GenericCard {...props} queryContext={queryContext} />
```

Comment:
The GenericCard did not have the queryContext (so the table entity check was failing).  Fixed by explicitly passing it down.